### PR TITLE
Misc.

### PR DESCRIPTION
**Pull request by fisx**
this shouldn't be too controversial.

### CODE REVIEW

**Comment by fisx** [comment]
ready to merge.  this time for real!  (-:

File: src/EventLog.hs [hunk 1]
```diff
@@ -93,7 +93,7 @@ instance CSV.ToRecord URLEventLogItem where
         objDesc (Left3   t) = "Thema " <> t ^. topicTitle . showed . csi
         objDesc (Middle3 i) = "Idee "  <> i ^. ideaTitle  . showed . csi
         objDesc (Right3  c) = "Verbesserungsvorschlag "
-                           <> ST.take 30 (c ^. commentText . showed . csi) <> "..."
+                           <> ST.take 30 (c ^. commentText . to unMarkdown . csi) <> "..."
```
Comment (np) [inline src/EventLog.hs]: `to unMarkdown` -> `_Markdown`

**Comment by andorp** [comment]
LGTM

**Comment by np** [comment]
I made one optical comment :) LGTM otherwise merge when you want.

**Comment by fisx** [comment]
Thanks, I'll merge when green.

**Pull request merged**